scene/resources/capsule_shape_3d.cpp

```diff
@@ -29,7 +29,7 @@
 /**************************************************************************/
 
 #include "capsule_shape_3d.h"
-
+#include "primitive_meshes.h"
```

Comment:
```suggestion

#include "scene/resources/primitive_meshes.h"
```
Same here, and keep a space